rust/src/runner/context.rs

```diff
@@ -19,7 +19,7 @@ use strum_macros::EnumIter;
 use tempfile::{tempdir_in, TempDir};
 use thiserror::Error;
 
-use crate::{config::SettingsConfig, test::TestError};
+use crate::{config::SettingsConfig, test::TestError, tests::lchmod};
```

Comment:
This creates a circular modules use graph.  That's confusing.  If lchmod needs to be used both here and in the tests, I suggest you put it in a "utils" module or something.

Reply:
Oh yes, sorry!